diagnostic.md

```diff
@@ -9,6 +9,12 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+primitive : example
+String : 'stuff'
+Number : 4
+boolean : true
+
```

Comment:
Not quite. Please review this lesson: https://github.com/ga-wdi-boston/js-basics